modules/tcfControl.js

```diff
@@ -367,8 +369,25 @@ export function setEnforcementConfig(config) {
   }
 }
 
+export function checkIfCredentialsAllowed(next, options = {}, moduleType, moduleName) {
+  if (!options.withCredentials || (moduleType && moduleName)) {
+    next(options);
+    return;
+  }
+  const consentData = gdprDataHandler.getConsentData();
+  const rule = ACTIVE_RULES.purpose[1];
+  const ruleOptions = CONFIGURABLE_RULES[rule.purpose];
+  const {purpose} = getConsent(consentData, ruleOptions.type, ruleOptions.id, null);
+  if (!purpose) {
+    options.withCredentials = false;
+    logWarn('withCredentials option overwritten with false accordingly to tcf consent data');
```

Comment:
```suggestion
    logWarn(${RULE_NAME} denied '${ACTIVITY_ACCESS_REQUEST_CREDENTIALS}'');
```

Reply:
for consistency with the "normal" case

these messages are meant for users (publishers), and I expect  "withCredentials options  overwritten with false" to mean nothing to them